src/main/java/seedu/address/logic/commands/AddContactToTeamCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+import seedu.address.model.team.Team;
+
+import static java.util.Objects.requireNonNull;
+
+public class AddContactToTeamCommand extends Command {
+
+    public static final String COMMAND_WORD = "add-to-team";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a contact to the team. "
+            + "Parameters: "
+            + "Team index number, contact index number \n"
+            + "Example: add-to-team 2 3";
+
+
+    public static final String MESSAGE_SUCCESS = "New contact added to team: %1$s";
+    public static final String MESSAGE_DUPLICATE_CONTACT_IN_TEAM = "A contact with this name already exists in " +
+            "the team";
+
+    private final int teamIndex;
+    private final int contactIndex;
+
+    /**
+     * Creates an AddContactToTeamCommand to add the specified {@code Contact}
+     */
+    public AddContactToTeamCommand(int teamIndex, int contactIndex) {
```

Comment:
Use `Index` instead of `int`, this way you don't have to worry about subtracting one to get the correct index.